packages/frontend/src/components/common/ChartDownload/chartConfigs.ts

```diff
@@ -0,0 +1,54 @@
+import { type EChartsInstance } from 'echarts-for-react';
+export const chartConfigs = (chartInstance: EChartsInstance) => {
+    const isPieChat = chartInstance._chartsViews.some(
+        (view: { constructor: { name: string } }) =>
+            view.constructor.name === 'PieView2',
```

Comment:
not sure if there's a better way to do this - perhaps from a prop in ChartDownloadOptions